plugins/kotlin-as-java/src/main/kotlin/transformers/JvmNameDocumentableTransformer.kt

```diff
@@ -55,33 +53,34 @@ class JvmNameDocumentableTransformer : DocumentableTransformer {
         return extraWithoutAnnotations.addAll(listOfNotNull(annotationsWithoutJvmName))
     }
 
-    private fun transformClassLike(documentable: DClasslike): DClasslike =
+    @Suppress("MemberVisibilityCanBePrivate")
+    fun transformClassLike(documentable: DClasslike, context: DokkaContext): DClasslike =
```

Comment:
Is it not better to make `transform` public?

Reply:
Yes, it will be more accurate, will change